bin/all_sky_search/pycbc_template_kde_calc

```diff
@@ -0,0 +1,147 @@
+#3dimensional_kde 
+#python3 pycbc_template_kde_calc.py --template-file /home/praveen.kumar/focused_search/kde/sbbh2.hdf --signal-file /home/praveen.kumar/focused_search/kde/3OGC_PEtable.txt_ifar_gt0.5.csv     --make-signal-kde    --verbose
```

Comment:
Can you remove this line, the help message which is generated by argparse should be sufficient to explain how to write a command line for the code.